common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -71,3 +115,28 @@ jobs:
           nodeVersion: 20.x
       # The publish script identifies any new packages not previously published and tags the build
       - template: ../templates/publish.yaml
+
+  - job: PostResult
+    dependsOn:
+    - CheckLinkedPR
```

Comment:
So theres a job in the imodel-native pipeline which runs nearly the same bash script, correct?

I also thought that if there was a linked PR we skipped itwinjs core build anyways. Or I guess this was never necessary because its PRs into imodel02 which would typically have a linked PR (and therefore no build reqs to merge into imodel02 on this repo). Is that the reason?

Reply:
Yeah we never skipped builds before because imodel02 PRs skipped by default.

And yes in imodel-native theres a similar bash script however much more complicated because that pipeline needs a lot more info. This just needs to know a pr was linked.